lib/board.rb

```diff
@@ -0,0 +1,96 @@
+class Board
+  attr_reader :cells
+
+  def initialize
+    @cells = {}
+  end
+
+  def generate(size)
+    #code based on 4 x 4 currently
```

Comment:
Before merging, we should make sure this works for a board bigger than 4 x 4

Reply:
Gotcha! And we should test for this as well.